### ISSUE TITLE
Aspect Ratio option not working

### ISSUE THREAD

**Issue by anantprsd5** [google/cameraview#152]
When trying to change the aspect ratio, the camera view doesn't work and a black screen comes in its place

**Comment by dk1206** [comment]
Hey Anantprsd5,

Are you setting one of the available Aspect Ratios for your mobile phone? Could you maybe also share some code with your issue.

**Comment by anantprsd5** [comment]
@dk1206 Yes. I am using the demo application of the camera view and when I click on the menu to change the aspect ratio, I am getting a black screen like the screenshot below.
![photo_2017-05-12_21-49-26](https://cloud.githubusercontent.com/assets/20367966/26009269/ae8211b0-3765-11e7-913c-8b5e74ed86a8.jpg)

**Comment by dk1206** [comment]
I cannot reproduce it with my phone, for me it works. But that is rather curious aspect ratio you have btw.

If you can edit your initial message, please add the also the Mobile phone model you are using and also write there the OS version.

**Comment by anantprsd5** [comment]
@dk1206 Done.

**Comment by aalices** [comment]
The same issue for Xiaomi Redmi 4A, Android: 6.0.1
